LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -149,6 +150,28 @@ private String valueOf(final Object obj) {
 
     @Override
     public ILabAccount createTempAccount(@NonNull final TempUserType tempUserType) throws LabApiException {
+        // Adding a second attempt here, api sometimes fails to create the temp user.
+        try {
+            return createTempAccountInternal(tempUserType);
+        } catch (LabApiException e){
+            if (e.getErrorCode().equals(LabError.FAILED_TO_CREATE_TEMP_USER)){
```

Comment:
This should make it safe from NPE during equals check

```suggestion
            if (LabError.FAILED_TO_CREATE_TEMP_USER.equals(e.getErrorCode())){
```